docs/UserGuide.adoc

```diff
@@ -18,160 +18,512 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+HireLah! is targeted to bring greater convenience to interviewers who
+can type quickly and prefer typing over mouse commands. Additionally, we
+included customised features in our desktop apps to help interviewers to
+manage interviews.
 
 == Quick Start
-
-.  Ensure you have Java `11` or above installed in your Computer.
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Ensure you have Java 1 or above installed in your Computer.
+.  Download the latest `HireLah.jar` link:{repoURL}/releases[here].
+.  Copy the file to the folder you want to use as the home folder for your HireLah!.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
 image::Ui.png[width="790"]
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
 
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
-* *`exit`* : exits the app
+== Features
```

Comment:
If you go look at the preview this isn't rendered properly (and it doesn't appear in the Table of Contents). All the sub sections are wrongly put under "2. Quick Start" Probably some formatting issue.